dataverse-webapp/src/main/webapp/dataset.xhtml

```diff
@@ -947,33 +948,40 @@
                 </p:dialog>
 
                 <p:dialog header="#{bundle['dataset.rejectBtn']}" widgetVar="sendBackToContributor" modal="true">
-                    <p>
-                        <h:outputFormat id="textBoxName" value="#{bundle['dataset.reject.messageBox.label']}" escape="false"/>
-                    </p>
-                    <p:inputTextarea id="returnReason"
-                                     rows="3"
-                                     value="#{DatasetPage.returnToAuthorReason}"
-                                     maxlength="255"
-                                     widgetVar="returnReason"
-                                     cols="70"
-                                     counter="display"
-                                     counterTemplate="{0} characters remaining."
-                                     autoResize="false"
-                    >
-                        <p:ajax event="keyup" delay="100" global="false" update="confirmSentForModificationButton" />
-                    </p:inputTextarea>
-                    <p>
-                        <h:outputText id="display"/>
-                    </p>
-                    <p:watermark for="returnReason" value="#{bundle['dataset.rejectWatermark']}"
-                                 id="returnReasonwatermark"/>
-
+                    <div class="form-group">
+                        <label for="returnReasonTextArea" class="col-sm-3 control-label">#{bundle['dataset.reject.messageBox.label']}
+                            <span class="glyphicon glyphicon-asterisk text-danger" title="#{bundle.requiredField}"/>
+                        </label>
+                        <div class="col-sm-9">
+                            <p:fragment id="returnReasonFragment">
+                                <p:inputTextarea id="returnReasonTextArea"
+                                                 rows="3"
+                                                 value="#{DatasetPage.returnToAuthorReason}"
+                                                 maxlength="255"
+                                                 cols="60"
+                                                 counter="display"
+                                                 counterTemplate="#{bundle['dataset.reject.messageBox.counter']}"
+                                                 autoResize="false"
+                                                 required="#{true}"
+                                                 requiredMessage="#{bundle['dataset.reject.enterReason']}"
+                                                 styleClass="form-control"
+                                >
+                                </p:inputTextarea>
```

Comment:
Can you check to be sure? When we try to invoke any action on dataset page (for example edit tags, license of files or deacession of dataset) nothing will happen. Because this input textarea will be blocking it on validation.